storm-core/src/clj/org/apache/storm/daemon/nimbus.clj

```diff
@@ -587,7 +586,12 @@
 (defn update-heartbeats! [nimbus storm-id all-executors existing-assignment]
   (log-debug "Updating heartbeats for " storm-id " " (pr-str all-executors))
   (let [storm-cluster-state (:storm-cluster-state nimbus)
-        executor-beats (.executor-beats storm-cluster-state storm-id (:executor->node+port existing-assignment))
+        executor-beats (let [executor-stats-java-map (.executorBeats storm-cluster-state storm-id (.get_executor_node_port (thriftify-assignment existing-assignment)))
+                             executor-stats-clojurify (clojurify-structure executor-stats-java-map)]
+                         (->> (dofor [[^ExecutorInfo executor-info ^ClusterWorkerHeartbeat cluster-worker-heartbeat] executor-stats-clojurify]
+                             {[(.get_task_start executor-info) (.get_task_end executor-info)] (clojurify-zk-worker-hb cluster-worker-heartbeat)})
```

Comment:
with reference #diff-3dc2ee75946dae2afc9de7d4e5487219R191
in earlier version,
the structure returned was {["task_start", "task_end"] : {time-secs, uptime, executor-stat}}
now the structure returned seems to be {["task_start", "task_end"] : {storm-id, executor-stats, time-secs, uptime-secs}}
It seems for each executor, clojurified cluster worker heart beat is being stored. can you confirm this is not the case?